components/prize_pool/wikis/overwatch/prize_pool_legacy_custom.lua

```diff
@@ -0,0 +1,49 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:PrizePool/Legacy/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local PrizePoolLegacy = Lua.import('Module:PrizePool/Legacy', {requireDevIfEnabled = true})
+
+local CustomLegacyPrizePool = {}
+
+-- Template entry point
+function CustomLegacyPrizePool.run()
+	return PrizePoolLegacy.run(CustomLegacyPrizePool)
+end
+
+function CustomLegacyPrizePool.customHeader(newArgs, data, header)
+	newArgs.prizesummary = header.prizenote and true or newArgs.prizesummary
+
+	local localCurrency = Variables.varDefault('currency')
+	if not newArgs.localcurrency and localCurrency then
+		newArgs.localcurrency = localCurrency
+		data.inputToId.localprize = 'localprize'
+	end
+
+	return newArgs
+end
+
+function CustomLegacyPrizePool.customSlot(newData, data, slot)
+	-- Remove points with only image
+	-- Table.filter doesn't work on Tables (only Arrays)... Let's use Table.map instead
+	newData = Table.map(newData, function(key, value)
+		if key == 'points1' or key == 'points2' or key == 'points3' then
```

Comment:
you sure it is `points1` and not `points`

Reply:
Template wise im not seeing any differences between this and the Valorant version (that this was based off of). The behavior should be the same as well

Reply:
![E61648E3-B24A-480A-8D89-38EDFE0C6677](https://user-images.githubusercontent.com/75081997/189535277-edca9c7d-6c32-48d6-9a77-16eadf572940.jpeg)
![08026219-D335-4199-9BDC-BE9950691EA6](https://user-images.githubusercontent.com/75081997/189535281-0f2dcc56-c68b-466e-8c57-fe0243dec664.jpeg)